pkg/scheduler/plugins/gang/gang.go

```diff
@@ -183,6 +183,12 @@ func (gp *gangPlugin) OnSessionClose(ssn *framework.Session) {
 			unScheduleJobCount++
 			metrics.RegisterJobRetries(job.Name)
 
+			reason := scheduling.NotEnoughResourcesReason
+			// if job is rejected to enqueue, record the reason with Unenqueueable, to distinguish from NotEnoughResources
+			if job.PodGroup.Status.Phase != scheduling.PodGroupInqueue {
+				reason = scheduling.UnEnqueueableReason
```

Comment:
What if enqueue&preempt actions are not enabled，and cluster quota not enough？

Reply:
If there is no enqueue action, podgroup's phase will be equalt to `PodGroupInqueue`,  set by `allocate` action.